app/src/main/java/edu/uc/mille5bm/smartnutrition/ui/main/MainFragment.kt

```diff
@@ -1,30 +1,38 @@
 package edu.uc.mille5bm.smartnutrition.ui.main
 
 import android.os.Bundle
+import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
 import edu.uc.mille5bm.smartnutrition.R
 import java.util.zip.Inflater
 
-class MainFragment {
+class MainFragment: Fragment(){
 
     companion object {
         fun newInstance() = MainFragment()
     }
 
     private lateinit var viewModel: MainViewModel
 
-    /***override fun onCreateView(
+    override fun onCreateView(
             inflater: LayoutInflater, container: ViewGroup?,
             savedInstanceState: Bundle?
     ): View {
         return inflater.inflate(R.layout.main_fragment, container, false)
     }
 
-    override fun onActivityCreated(savedInstaceState: Bundle?){
+    /** override fun onActivityCreated(savedInstaceState: Bundle?){
         super.onActivityCreated(savedInstaceState)
-        viewModel = viewModelProvider.of(this).get(MainViewModel::class)
+        viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
+        viewModel.foods.observe(viewLifecycleOwner, Observer{
+                foods -> actFood.setAdapter(ArrayAdapter(context!!, R.layout.support_simple_spinner_dropdown_item, foods))
+        })
+        viewModel.fetchFoods()
 
-    }***/
+    } **/
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.